_lectures/lect12.md

```diff
@@ -0,0 +1,96 @@
+---
+num: "Lecture 12"
+desc: "Linked List and Ordered Linked List"
+ready: true
+lecture_date: 2024-05-09 15:30:00 -0700
+---
+
+[Slides folder]({{ site.slides_url }})
+
+[Recorded video]()
+
+# Plan for today
+Iclicker review of last class
+Linked list vs ordered linked list
+How to delete an item from linked list
+How to add an item to an ordered linked list
+
+# Iclicker Questions
+Question 1:
+Which of the following assertions is False
+A linked list is...
+A.
+B.
+C.
+D.
+E.
+
+Question 2:
+Given a linked list, which attribute will it have as part of its class definition?
+A. next
+B. data
+*C. head*
+D. self
+E. None of the above
+
+Question 3:
+Given a node that will be part of a linked list, which attribute will it not have as part of its class definition?
+A. next
+B. data
+C. head
+D. self
+E. None of the above
+
+Question 4:
+Given the following lines of code, what order do they need to go in to correctly add a new node to the list?
+
+Question 5:
+Which operator do we need to overload to implement object comparison as follows?
+
+# Linked List versus Ordered Linked List
+(Unordered) Linked Lists
+-The position of the nodes did not matter with respect to each other
+-Inserted at the start of the list
+
+Ordered Linked List is similar to Unordered Linked List
+-Except the nodes in the list are ordered with respect to each other
+-Need to overload the comparison operators
+-Adding the node requires us to put a node in the correct position
+
+# Note
+- Add practice for overloading operators 
```

Comment:
Is this a TODO for me? If so, can you please add the tag, so that I can quickly update the files by searching for it?